data_structures/src/superblock.rs

```diff
@@ -319,6 +377,25 @@ impl SuperBlockState {
         }
     }
 
+    /// Since we do not check signatures here, a superblock vote is valid if the signing identity
```

Comment:
I think the comment needs to be updated, at least i dont really understand what  is valid in this function